### ISSUE TITLE
TypeError on right-click

### ISSUE THREAD

**Issue by NielsNet** [ckeditor/ckeditor-dev#1181]
## Are you reporting a feature request or a bug?

Bug

## Provide detailed reproduction steps (if any)

1. Open a page with a CKEditor ([SDK Sample](https://sdk.ckeditor.com/samples/readonly.html))
2. Make it readonly without focussing the editor first
3. Right-click on a link

### Expected result

Nothing

### Actual result

A typeerror is thrown in the browser-console:
```
ckeditor.js:664 Uncaught TypeError: Cannot read property 'checkReadOnly' of undefined
    at Array.<anonymous> (ckeditor.js:664)
    at $.onShow (ckeditor.js:723)
    at Object.onShow (ckeditor.js:29)
    at $.show (ckeditor.js:725)
    at $.open (ckeditor.js:735)
    at $.<anonymous> (ckeditor.js:734)
    at ckeditor.js:27
```

## Other details

* Browser: Opera, Chrome<s>, MS Edge</s>
* OS: Windows 10
* CKEditor version: 4.7.3
* Installed CKEditor plugins: undefined

**Comment by msamsel** [comment]
I was able to reproduce it on Chrome 62 Win7.
Problem wasn't present on Edge and Chrome on MacOS.

**Comment by mlewand** [comment]
I can also repro it with vanilla CKEditor 4 on http://cdn.ckeditor.com/4.8.0/full/samples/old/readonly.html.

Looks like a regression introduced in 4.4.6.

Fails in http://cdn.ckeditor.com/4.4.6/full/samples/readonly.html

Works on http://cdn.ckeditor.com/4.4.5/full/samples/readonly.html

**Comment by jacekbogdanski** [comment]
With 4.4.6 version (https://github.com/ckeditor/ckeditor-dev/commit/3048943550530ba2f48b43ce48939184118b) we moved initial selection fix from the selection constructor to editable. With this change initial selection fix is executed on the editor `focus` event, probably removed or replaced by another `focus` event listener.

**Comment by mlewand** [comment]
Fix has been merged to `next` branch, and will be released in CKEditor 4.10.2.

You can already take a peek at it in our CKEditor preview builds. The simplest way would be to reuse our [legacy read-only sample](https://ckeditor4-preview.ckeditor.com/next/samples/old/readonly.html) - be sure to press "Make it read-only" button, and then right click the link without focusing the editor.
